### ISSUE TITLE
Binary agent not picking up additional disks

### ISSUE THREAD

**Issue by ghost** [henrygd/beszel#307]
Using the binary agent on my Proxmox VE as well as Backup servers, it won't pick up and monitor additional disks, just the root filesystem.

As documented, I used lsblk to identify the drives, added them to the beszel-agent.service, reloaded the service-daemon as well as restarting the agent-service, restarted the beszel-hub and even re-added the host - still nothing.

See screenshots below.

![lsblk](https://github.com/user-attachments/assets/c7d97e05-eb1a-4931-aafa-280ee17beb9f)

![beszel-agent service](https://github.com/user-attachments/assets/de585a64-23be-4865-b3d5-ce754b5ab8ab)

**Comment by henrygd** [comment]
Please restart the agent with `LOG_LEVEL=debug` and let me know what it outputs for partitions and diskstats.

**Comment by henrygd** [comment]
Thanks, there should be a couple lines that are logged once, immediately after startup, and begin with `DEBUG Disk partitions` and `DEBUG Disk I/O diskstats`. Can you provide those lines please?

**Comment by henrygd** [comment]
Okay, try `EXTRA_FILESYSTEMS=sda1` or even use the mount point: `EXTRA_FILESYSTEMS=/mnt/datastore/pbs01`

You can reference the `DEBUG Disk partitions=` line to see which partitions are available to monitor.

In this case, you have three partitions with mount points, so those are the ones that the agent can see. 

- `sda1` - which is mounted at `/mnt/datastore/pbs01`
- `nvme0n1p2` - your boot partition mounted at `/boot/efi` - probably no need to monitor this
- `dm-1` which is mapped to your root partition and being picked up automatically.

**Comment by henrygd** [comment]
Correct, we're using gopsutil's [disk.Usage](https://pkg.go.dev/github.com/shirou/gopsutil/v4@v4.24.10/disk#Usage), which requires a mount point to calculate usage via the filesystem.

Maybe in the future we can attempt to read block counts if there's no mount point, but I haven't looked into it yet to see if that's feasible.

For ceph specifically, there appears to be some utility commands that give that info, like `sudo ceph-volume lvm list`. If there's a command that provides the usage and doesn't require root then I'd be open to adding support for it in Beszel.

**Comment by Martin-Fox** [comment]
@Turney1337 
How did you install binary agent in proxmox? Manually, or using script?
I'm asking, because in my case, this service was removed automatically by proxmox.

**Comment by fett1979** [comment]
Same problem here :)
Promox user!!!

**Comment by svenvg93** [comment]
No activity on this issue for a long time. Closing it for now. If the issue persists or becomes relevant again, feel free to open a new issue or discussion.
